src/main/resources/com/sonar/sqale/checkstyle-model.xml

```diff
@@ -894,6 +894,19 @@
           <txt>min</txt>
         </prop>
       </chc>
+      <chc>
+        <rule-repo>checkstyle</rule-repo>
+        <rule-key>com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocTypeCheck</rule-key>
```

Comment:
@muhlba91 Does every module need to be listed here?
It doesn't seem to be part of our test to verify everything is listed in here. https://github.com/checkstyle/sonar-checkstyle/blob/master/src/test/java/org/sonar/plugins/checkstyle/internal/ChecksTest.java#L87-L94

Reply:
I don't use SQALE on my instances but I don't think it's necessary since, afaik, if it's not listed iit doesn't have an impact on the score?
However, I added it because there were other Javadoc checks present and I kept it on-par with them. Do we have any rules for the SQALE stuff?

Anyhow, how is SQALE the SQALE support in SonarQube atm? At least in the API I read that it got deprecated:
- http://javadocs.sonarsource.org/7.7/apidocs/deprecated-list.html and search for SQALE
- https://jira.sonarsource.com/browse/SQALE-277?jql=project%20%3D%20SQALE -> the project is named "[DEPRECATED] SQALE"

Reply:
This is my first time seeing it so I don't believe there are any rules. I don't know what SQALE is so I couldn't identify what needs to be done with it.
Unless you or @romani know more, I suggest we just create an issue for a test to verify it has valid information and not require checks be listed.

Reply:
this config came from Sonarqube team.
I never understand how to it is possible to estimate fix in time, in most cases it does work at all. 
Such estimations NEVER worked well, it was only source of laugh ..... ones you spend a day with "10 min fix".
If Sonar do not demand such config be for each rule - I would be happy to remove them completely.

Reply:
ah ok, I'll see if Sonar is happy when the file is empty/removed. I think the listed checks are anyways not up to date with all checks available.
If so, we can remove it and also don't have to copy over the SQALE source code from sslr. Will create an issue on that latest tomorrow. :)

Reply:
created #219